doc/source/cluster/kubernetes/troubleshooting/troubleshooting.md

```diff
@@ -7,13 +7,21 @@ If you don't find an answer to your question here, please don't hesitate to conn
 
 # Contents
 
+- [Use ARM-based docker images for Apple M1/M2 Macbooks](#docker-image-for-apple-macbooks) 
 - [Upgrade KubeRay](#upgrade-kuberay)
 - [Worker init container](#worker-init-container)
 - [Cluster domain](#cluster-domain)
 - [RayService](#rayservice)
 - [Autoscaler](#autoscaler)
 - [Other questions](#other-questions)
 
+(docker-image-for-apple-macbooks)=
+## Use ARM-based docker images for Apple M1/M2 Macbooks
```

Comment:
not sure we even want to support this. we never really tested kuberay on arm..

Reply:
We haven't tested KubeRay on ARM in CI, but KubeRay builds a multi-architecture image that supports both `linux/amd64` and `linux/arm64`. We conducted some CUJ analyses for the KubeRay documentation and found that almost all users struggle to run an easy KubeRay example on Mac M1 because they need to update to an ARM64-compatible Ray image. That's why I ask @shaikhi to open this PR. Thank @aslonnie for the review!

Reply:
does kuberay prefer to have multi-arch ray images? we can build those, it is not actually not very hard.

the more tricky / expensive part is testing. like what kind of quality / reliability bar we want to reach for this?

Reply:
> does kuberay prefer to have multi-arch ray images? 

It would be very helpful for UX! The multi-architecture image can increase the success rate for users following the KubeRay documentation. We only need to ensure that users can easily follow the documents with the multi-architecture image, and we still recommend that users use the original images for production.

Reply:
if we decide to build them and **not** test them we need to be explicit in documentation on docs.ray.io, and github. who can take that action item to update?

Reply:
I think only this comment/note is unresolved from @anyscalesam :
> if we decide to build them and not test them we need to be explicit in documentation on docs.ray.io, and github. who can take that action item to update?

I'm not sure what to do for this? Can we create another ticket for this?

Reply:
@shaikhismail I will take that part after the multi-platform images are out. I think the PR is ready to merge.